scarlet2/morphology.py

```diff
@@ -37,12 +37,18 @@ class ProfileMorphology(Morphology):
     center: jnp.array
     size: float
     ellipticity: (None, jnp.array)
+    g: (None, jnp.array)
```

Comment:
Can you implement `g` as a class property instead of a class member, please. It's a quick computation, so we can avoid the redundancy of the definitions.

Reply:
Sure, done.